examples/wordpress/README.md

```diff
@@ -0,0 +1,154 @@
+# How To Use NFS Persistent Volumes
+
+The purpose of this guide is to help you understand storage provisioning with NFS by creating a WordPress blog and MySQL database.
+Both the blog and database require persistent storage.  
+
+This guide assumes knowledge of OpenShift fundamentals and that you have a cluster up and running.  Please review steps 1 - 10 in the
+[sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md) to run an OpenShift cluster.
+
+## NFS Provisioning
+
+We'll be creating NFS exports on the local machine.  The instructions below are from Fedora.  The provisioning process may be slightly different based on linux distribution or the type of NFS server being used.
+
+Create two NFS exports, each of which will become a Persistent Volume in the cluster. 
+
+```
+# the directories in this example can grow unbounded
+# use disk partitions of specific sizes to enforce storage quotas
+mkdir /home/data/pv0001 
+mkdir /home/data/pv0002
+
+# data written to NFS by a pod gets squashed by NFS and is owned by 'nfsnobody'
+# we'll make our export directories owned by the same user
+chown -R /home/data nfsnobody:nfsnobody
+
+# security needs to be permissive currently, but the export will soon be restricted 
+# to the same UID/GID that wrote the data
+chmod -R 777 /home/data/
+
+# Add to /etc/exports
+/home/data/pv0001 *(rw,sync,no_root_squash)
+/home/data/pv0002 *(rw,sync,no_root_squash)
+
+# start NFS or restart to pick up the new exports
+systemctl restart nfs.service
```

Comment:
`exportfs -a` is the method enable new exports without bouncing the NFS service.